<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orakel8</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressourcen.qrc">
    <normaloff>:/bilder/Kugel8.png</normaloff>:/bilder/Kugel8.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 190, 190);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Sch√ºtteln</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBild">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelKugel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>labelBild</zorder>
   <zorder>pushButton</zorder>
   <zorder>labelKugel</zorder>
  </widget>
 </widget>
 <resources>
  <include location="ressourcen.qrc"/>
 </resources>
 <connections/>
</ui>
